[2024-10-28T20:54:35.449+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-10-28T20:54:35.463+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: tab_refresh_stats.daily_instagram_stats manual__2024-10-28T20:54:34.164128+00:00 [queued]>
[2024-10-28T20:54:35.468+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: tab_refresh_stats.daily_instagram_stats manual__2024-10-28T20:54:34.164128+00:00 [queued]>
[2024-10-28T20:54:35.468+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 2
[2024-10-28T20:54:35.477+0000] {taskinstance.py:2888} INFO - Executing <Task(PythonOperator): daily_instagram_stats> on 2024-10-28 20:54:34.164128+00:00
[2024-10-28T20:54:35.485+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=302) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-10-28T20:54:35.486+0000] {standard_task_runner.py:72} INFO - Started process 304 to run task
[2024-10-28T20:54:35.487+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'tab_refresh_stats', 'daily_instagram_stats', 'manual__2024-10-28T20:54:34.164128+00:00', '--job-id', '10', '--raw', '--subdir', 'DAGS_FOLDER/tab_social_media_refresh.py', '--cfg-path', '/tmp/tmp62el63q1']
[2024-10-28T20:54:35.488+0000] {standard_task_runner.py:105} INFO - Job 10: Subtask daily_instagram_stats
[2024-10-28T20:54:35.522+0000] {task_command.py:467} INFO - Running <TaskInstance: tab_refresh_stats.daily_instagram_stats manual__2024-10-28T20:54:34.164128+00:00 [running]> on host 721aaaeb5415
[2024-10-28T20:54:35.810+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='daniel.cristancho' AIRFLOW_CTX_DAG_ID='tab_refresh_stats' AIRFLOW_CTX_TASK_ID='daily_instagram_stats' AIRFLOW_CTX_EXECUTION_DATE='2024-10-28T20:54:34.164128+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-10-28T20:54:34.164128+00:00'
[2024-10-28T20:54:35.811+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-10-28T20:54:37.562+0000] {evolution_stats_instagram.py:35} INFO - Start extract loaded data
[2024-10-28T20:54:43.567+0000] {evolution_stats_instagram.py:38} INFO - Start load new data
[2024-10-28T20:54:43.568+0000] {evolution_stats_instagram.py:108} INFO - writing user_follow_stats table
[2024-10-28T20:54:45.802+0000] {evolution_stats_instagram.py:112} INFO - 4 rows load in user_follow_stats
[2024-10-28T20:54:45.803+0000] {evolution_stats_instagram.py:113} INFO - loading operation finished successfully for user_follow_stats
[2024-10-28T20:54:45.803+0000] {logging_mixin.py:190} INFO - ****************** Success process to update instagram post
[2024-10-28T20:54:45.803+0000] {python.py:240} INFO - Done. Returned value was: None
[2024-10-28T20:54:45.813+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-10-28T20:54:45.813+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=tab_refresh_stats, task_id=daily_instagram_stats, run_id=manual__2024-10-28T20:54:34.164128+00:00, execution_date=20241028T205434, start_date=20241028T205435, end_date=20241028T205445
[2024-10-28T20:54:45.876+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-10-28T20:54:45.887+0000] {taskinstance.py:3900} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-10-28T20:54:45.888+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
